src/main/kotlin/de/bund/digitalservice/useid/widget/WidgetController.kt

```diff
@@ -66,7 +66,7 @@ class WidgetController(
         )
 
         if (isIncompatibleOSVersion(userAgent)) {
```

Comment:
This condition should be moved to the top of the function right? So that only one tracking event is published (either "widget loaded" or "incompatible").

Reply:
we discussed this today and we will keep it as it is, since we want to see the user journey from widget loaded to incompatible page
cc: @maximilianwoltersdorf